ToDoWebApi/Applications/UserOperations/Commands/UpdateUser/UpdateUserCommand.cs

```diff
@@ -4,37 +4,41 @@ namespace ToDoWebApi.Applications.UserOperations.UpdateUser
 {
     public class UpdateUserCommand
     {
-        public const string ExceptionMessageFound = "User does not exist.";
+        public const string ExceptionMessageNotFound = "User does not exist.";
         public const string ExceptionMessageEmail = "User E-Mail is wrong.";
         public const string ExceptionMessagePassword = "User Password is wrong.";
 
-
-        public UpdateUserModel Model { get; set; }
-        public int Id { get; set; }
+        public UpdateUserViewModel Model { get; set; }
 
         private readonly IToDoDbContext _context;
 
         public UpdateUserCommand(IToDoDbContext context, int id)
         {
             _context = context;
-            Id = id;
+            Model.Id = id;
```

Comment:
Böyle bir şey yok. Bu command'in contructor'ında bu Model daha boş. Id,yi contructor,dan kaldırıcaksın tamamen, modelin nerede doluyorsa orada yapıcaksın. Önceden de aynı hatayı yapmıştın.

Reply:
Düzeltildi.